src/org/thoughtcrime/securesms/ProfileActivity.java

```diff
@@ -385,13 +385,16 @@ public void onSoundSettings() {
   }
 
   public void onVibrateSettings() {
+    int checkedItem = Prefs.getChatVibrate(this, chatId).getId();
+    int[] selectedChoice = new int[]{checkedItem};
     new AlertDialog.Builder(this)
```

Comment:
Can we change this? It should be a function instead.

Reply:
Nevermind, I thought this was JavaScript ;)

Reply:
Just .. ignore me. It's for the best :wink:

Reply:
:)